bin/commands.js

```diff
@@ -19,10 +19,14 @@ const libraries = {
 };
 
 const help = `
-    ${chalk.whiteBright("--discord")}       : Shows discord support server invite
-    ${chalk.whiteBright("--help")}          : Shows help menu
-    ${chalk.whiteBright("--version")}       : Shows CDA version
-    ${chalk.whiteBright("--create")}        : Create Project
+    ${chalk.whiteBright("--discord")}       : Shows discord support server invite.
+    ${chalk.whiteBright("--help")}          : Shows help menu.
+    ${chalk.whiteBright("--version")}       : Shows CDA version.
+    ${chalk.whiteBright("--create")}        : Create Project.
+
+    Subcommands:
```

Comment:
could you add `--dir` to the sub-commands?

Reply:
Yeah added `--force` sub command too.